opentelemetry-jaeger/src/exporter/config/collector/mod.rs

```diff
@@ -165,15 +173,33 @@ impl Default for ClientConfig {
 ///
 /// See details for each configurations at [`CollectorPipeline`].
 ///
+/// Deprecation Notice:
+/// Ingestion of OTLP is now supported in Jaeger please check [crates.io] for more details.
+///
 /// [`CollectorPipeline`]: crate::config::collector::CollectorPipeline
+/// [crates.io]: https://crates.io/crates/opentelemetry-jaeger
 #[cfg(feature = "collector_client")]
+#[deprecated(
+    since = "0.21.0",
+    note = "Please Migrate to opentelemetry-otlp exporter."
+)]
 pub fn new_collector_pipeline() -> CollectorPipeline {
     CollectorPipeline::default()
 }
 
 /// Similar to [`new_collector_pipeline`] but the exporter is configured to run with wasm.
+///
+/// Deprecation Notice:
+/// Ingestion of OTLP is now supported in Jaeger please check [crates.io] for more details.
+///
+/// [crates.io]: https://crates.io/crates/opentelemetry-jaeger
 #[cfg(feature = "wasm_collector_client")]
-#[allow(clippy::field_reassign_with_default)] // make sure when collector_cilent and wasm_collector_client are both set. We will create a wasm type client
+#[allow(clippy::field_reassign_with_default)]
+// make sure when collector_cilent and wasm_collector_client are both set. We will create a wasm type client
+#[deprecated(
+    since = "0.21.0",
+    note = "Please Migrate to opentelemetry-otlp exporter."
```

Comment:
wasm is not available for now. We probably will have to stablize the existing API before explore the wasm support

Reply:
So is this the reason people are still using Jaeger? if it's WASM then that seems like a blocker to me.

Reply:
It could be? But crates.io doesn't provide feature level metrics so it's hard to say for sure.